components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -476,6 +487,32 @@ && isStepHasMultiOption(context)
         }
     }
 
+    private boolean checkIfApplicationAccessEnabled(HttpServletRequest request, CommonAuthResponseWrapper responseWrapper,
+                                                 AuthenticationContext context) throws IOException, FrameworkException {
+
+        String type = request.getParameter(TYPE);
+        String clientId = request.getParameter(CLIENT_ID_IN_REQUEST);
```

Comment:
```suggestion
        String clientId = request.getParameter(FrameworkConstants.RequestParams.ISSUER);
```
better if we can use `request.getParameter(FrameworkConstants.RequestParams.ISSUER);` as that is what we have done in other places. Is there any reason to not rely on the ISSUER?

Reply:
I have not observed we have relying party in the inital request but the client ID. After context is initiated I was able to fetch the relying party from context.